Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MasterDetailsView/MasterDetailsView.bind

```diff
@@ -0,0 +1,36 @@
+﻿<Page
+    x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MasterDetailsView.MasterDetailsViewPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:models="using:Microsoft.Toolkit.Uwp.SampleApp.Models"
+    mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+      <controls:MasterDetailsView x:Name="MasterDetails" ItemsSource="{x:Bind Emails}"
+                                  Foreground="Black">
+        <controls:MasterDetailsView.ItemTemplate>
+          <DataTemplate x:DataType="models:Email">
+            <StackPanel Margin="8">
+              <TextBlock Text="{x:Bind From}" Style="{ThemeResource SubtitleTextBlockStyle}"/>
+              <TextBlock Text="{x:Bind Subject}" Style="{ThemeResource BodyTextBlockStyle}" Foreground="{ThemeResource Brush-Blue-01}" MaxLines="1"/>
+              <TextBlock Text="{x:Bind Body}" Style="{ThemeResource BodyTextBlockStyle}" Opacity=".6" MaxLines="1"/>
+            </StackPanel>
+          </DataTemplate>
+        </controls:MasterDetailsView.ItemTemplate>
+        <controls:MasterDetailsView.DetailsTemplate>
+          <DataTemplate>
+            <RelativePanel Margin="24">
+              <Ellipse x:Name="FromEllipse" Height="50" Width="50" Fill="Gray"></Ellipse>
+              <TextBlock Text="{Binding From}" Style="{ThemeResource SubtitleTextBlockStyle}" RelativePanel.RightOf="FromEllipse" Margin="12,-6,0,0"/>
+              <TextBlock x:Name="SubjectLine" Text="{Binding Subject}" Style="{ThemeResource BodyTextBlockStyle}" RelativePanel.Below="FromEllipse"/>
+              <TextBlock x:Name="Body" Text="{Binding Body}" Style="{ThemeResource BodyTextBlockStyle}" TextWrapping="Wrap" RelativePanel.Below="SubjectLine" Margin="0,12,0,0"/>
+              <Button Content="Click ME!" RelativePanel.Below="Body"/>
```

Comment:
the purpose is to click it ;)
I think I was going to add reply/forward/etc buttons but decided against it. Will be removed